### PR TITLE
Js 2017

### PR DESCRIPTION
**Pull request by OmgImAlexis**
@p0psicles

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    'static/js/lib/**/*',
+    'static/js/vender.min.js',
+    'static/js/api.js',
+    'static/css/lib/**/*',
+    'static/css/*.ttf'
+];
+
+gulp.task('default', ['build']);
+
+gulp.task('build', done => {
```
[the remark this answers is not in the document]
Comment (p0psicles) [reply gulpfile.js]: Strange warnings. Parentheses are not needed at all with arrow notation and single var.

[the remark this answers is not in the document]
Comment (OmgImAlexis) [reply gulpfile.js]: I thought we disabled codacy for the js files?

**Comment by OmgImAlexis** [comment]
@medariox @p0psicles if one of you could review this and make sure pages are still working then we can merge this.

**Comment by p0psicles** [comment]
Still issues with connecting to the websocket.
Noticed it with and without a webroot.

with webroot:
```
WebSocket connection to 'ws://localhost:8081/ws/ui' failed: Error during WebSocket handshake: Unexpected response code: 404
```

Without webroot:
```
WebSocket connection to 'ws://localhost:8081//ws/ui' failed: Error during WebSocket handshake: Unexpected response code: 404
```

**Comment by OmgImAlexis** [comment]
Waiting on https://github.com/niftylettuce/frisbee/pull/53

**Comment by OmgImAlexis** [comment]
Waiting on https://github.com/niftylettuce/frisbee/pull/54

**Comment by p0psicles** [comment]
Can we do some thing about the codacy and Travis warnings?

**Comment by OmgImAlexis** [comment]
@p0psicles I can't see any major issues. Want to review?

**Comment by p0psicles** [comment]
Biggest issue is the config route being loaded multiple times.
That really needs to be fixed.

image

**Comment by p0psicles** [comment]
For babel I think we need to add a preset, like babel-preset-env.

**Comment by OmgImAlexis** [comment]
I can fix the config issue after the review. I mainly wanted this all checked before moving more files around.

**Comment by medariox** [comment]
_Windows 10, Chrome 63_, `medusa` as web_root

While browsing any page:
img

While browsing any show page (displayShow):
img2

nav

Websockets:
webs

Manage -> subtitles missed:
"Expand" not working
subs

**Comment by p0psicles** [comment]
I'm okay with the setup. But as discussed allot of it will need to move again when going to the themed setup. Let's continue with this now.
We can try to fix the IE11 stuff after this is all finished. So let's keep developing now using ES6. And revert the stuf for IE11 later.

**Comment by OmgImAlexis** [comment]
@p0psicles after we get your PR merged how shall we go about fixing this?

**Pull request closed without merging**